packages/client/src/client/utils/window/openWindow.web.ts

```diff
@@ -59,16 +57,9 @@ export function openWindow(
   return Promise.resolve(win || undefined)
 }
 
-function calculatePosition(
-  center = "parent",
-  width: number,
-  height: number,
-  reference?: { top: number; left: number },
-) {
+function calculatePosition(center = "parent", width: number, height: number) {
```

Comment:
this was two functions in one .. extracted the reference-win-based stuff as it is just a simple expression, leaving the parent/screen-based calcs in this function

Reply:
.. and simplified, according to usage